lib/customer.rb

```diff
@@ -0,0 +1,44 @@
+require_relative "dishes"
+
+class Customer
+require 'twilio-ruby'
+
+attr_reader :customer_dishes, :order_msg
+
+  def initialize
+    @customer_dishes = []
+    @order_msg = "Thank you! Your order was placed and will be delivered"
+  end
+
+  def check
+    @dishes
+  end
+
+  def select(dish)
+    @customer_dishes << dish
+  end
+
+  def amount_of_dishes
+    @customer_dishes.count
+  end
+
+#----- I HAVE OUTPUT THE TEXT MESSAGE TO THE TERMINAL -----
```

Comment:
It's great that you got the text message working 👍

Reply:
Thanks Josh it was fun and rewarding to do 👍